### DISCUSSION TITLE
Блокування copilot на території України

### DISCUSSION THREAD

**Discussion by Arkl1ne** [community/community#47645] [Copilot Conversations]
По якій причині доступ до GitHub copilot заблокували на території України? У мене у голові є тільки одна думка, чому ви могли так зробити. Це те, що ви підтримуєте дії окупанта (росії). Якщо ні, тоді чому заблокували доступ?

**Comment by vadimfdit** [comment]
Теж цікаво, при тому що я наприклад заплатив гроші за рік підписки 😡

**Reply by Arkl1ne** [reply]
З VPN якщо що працює. VPN UNLIMITED дає безкоштовну підписку на рік для України

**Reply by ShouldRa** [reply]
Шо за фігня? я заплатив гроші

**Comment by tuves** [comment]
Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect.

Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.

We sincerely apologize for any stress and/or frustration this has caused.

Additionally, we take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
